src/actions/walletActions.js

```diff
@@ -343,23 +343,25 @@ function clearDecryptCloudWalletError () {
 }
 
 async function _getLastUsedAddress (idToken: string) {
-  let response = await API.getLastUsedAddress(idToken)
+  let response = await API.getLastUsedAddress({ idToken })
   let rv = {}
   const walletTypeList = ['drive', 'metamask', 'ledger']
   const cryptoTypeList = ['bitcoin', 'ethereum', 'dai', 'libra']
   // convert response to our wallet struct
-  walletTypeList.forEach((walletType) => {
-    if (response[walletType]) {
-      rv[walletType] = { crypto: {} }
-      cryptoTypeList.forEach((cryptoType) => {
-        if (response[walletType].crypto && response[walletType].crypto[cryptoType]) {
-          rv[walletType].crypto[cryptoType] = [
-            response[walletType].crypto[cryptoType]
-          ]
-        }
-      })
-    }
-  })
+  if (response) {
+    walletTypeList.forEach((walletType) => {
+      if (response[walletType]) {
+        rv[walletType] = { crypto: {} }
+        cryptoTypeList.forEach((cryptoType) => {
+          if (response[walletType].crypto && response[walletType].crypto[cryptoType]) {
+            rv[walletType].crypto[cryptoType] = [
+              response[walletType].crypto[cryptoType]
```

Comment:
this part needs to be updated, rv does not contain correct lastUesedWallet info

Reply:
fixed